server/internal/infrastructure/mongo/client.go

```
@@ -0,0 +1,49 @@
+package mongo
+
+import (
+	"context"
+	"time"
+
+	"github.com/dont-wait/anomaly/internal/domain"
+	"github.com/dont-wait/anomaly/internal/logger"
+	"github.com/rs/zerolog"
+	"go.mongodb.org/mongo-driver/v2/mongo"
```

golangci-lint (2.12.2)

[error] ST1019: package "go.mongodb.org/mongo-driver/v2/mongo" is being imported more than once

(staticcheck)